### DISCUSSION TITLE
Is there any way to use +/-?

### DISCUSSION THREAD

**Discussion by kud** [iSoron/uhabits#1588] [Help & FAQ]
Hey,

Imagine I want to reduce the number of cups of coffee every day, like max 2 times.

Is there a way to use a + to increment the number? For the moment I only see a numeric keyboard to do it and yes I'm so lazy, it's a deal breaker for me regarding the UX.

When I do a habit, I just want to click on "plus" or "-" because yeah I don't want to have any mental load, to count +1 in my head, haha.

Thanks

**Comment by iSoron** [comment]
Hi @kud, that's not currently possible.
